Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -39,27 +39,49 @@ public async Task<bool> Execute(CreateUserRequest request)
             validator.ValidateAndThrow(request);
 
             string passwordHash = ShaHash.GetPasswordHash(request.Password);
-
-            var user = new User
+            User user;
+            UserAvatar userAvatar = null;
+            if (request.Avatar.Length == 0)
             {
-                Id = Guid.NewGuid(),
-                Birthday = request.Birthday,
-                FirstName = request.FirstName,
-                LastName = request.LastName
-            };
-
+                user = new User
+                {
+                    Id = Guid.NewGuid(),
+                    Birthday = request.Birthday,
+                    FirstName = request.FirstName,
+                    LastName = request.LastName

```

Comment:
why you cannot set these parameters when creating

Reply:
I didn't understand you. Could you explain in more detail?

Reply:
you can write like this:
var user = new User 
{
                    Id = Guid.NewGuid(),
                    Birthday = request.Birthday,
                    FirstName = request.FirstName,
                    LastName = request.LastName
} 
when you create user for the first time